temba/settings_common.py

```diff
@@ -285,20 +285,20 @@
 BRANDING = {
     "rapidpro.io": {
         "slug": "rapidpro",
-        "name": "RapidPro",
```

Comment:
You can keep the original code on this file, we are changing those data via environment variables on https://github.com/greatnonprofits-nfp/rapidpro/blob/production-2.0/docker/docker.settings

Reply:
Got it. I've reverted this file.